src/licensedcode/data/rules/python_98.RULE

```diff
@@ -0,0 +1,12 @@
+---
+license_expression: python
+is_license_notice: yes
+referenced_filenames:
+    - LICENSE
+---
+
```

Comment:
We should avoid this empty line everywhere.

Reply:
I think this is a wart in yaml frontmatter processing or how we do the processing, related to whitespace preceding rules/licenses, I'll link here a previous discussion on this, but this is not reflected in the actual loaded License/Rule as it is stripped. See https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/apache-2.0.LICENSE#L28 and then the text for https://scancode-licensedb.aboutcode.org/apache-2.0.html and this empty line is not included here, so we're good.